students/Lincoln_Zhang/Session5/mailroom_part3.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+import sys,os,textwrap
+
+print("Mailroom")
+
+donors_db = {"Fred Flintstone":[100, 200, 50],
+          "Barney Rubble":[100, 50, 600],
+          "Wilma Flintstone":[1000, 50, 600, 200],
+          "Pebbles Flintstone":[10, 5, 6],
+          }
+
+
+def gen_letter(name, amount):
+    letter = textwrap.dedent("""
+            Dear {},
+
+            Thank you! The amount you donated is ${:.2f}!
+
+            Sincerely
+            The Team
+
+            """.format(name,amount))
+    return letter
+
+
+
+def Mail_to_donors(a_dict):
+    """
+    This Function is to generate Thank you letter to all donors in donors_db
+
+    and store the letters in letters.txt file
+    """
+    with open ("letters.txt","w+") as f:
+        for item in a_dict.items():
```

Comment:
you can rewrite this as:
for name, amount in a_dict: (this helps you get rid of the next line)

Reply:
actually is 
for name,amount in a_dict.items():
otherwise I got error "ValueError: too many values to unpack (expected 2)"